### ISSUE TITLE
[DPG] Full CADL-Ranch coverage

### ISSUE THREAD

**Issue by lmazuel** [Azure/autorest.java#2057]
Title is self explanatory :)

**Comment by haolingdong-msft** [comment]
Note: One blocker I can see to reach to 100% coverage is that some cases not applicable to specific language, like model Inheritance for .net and java, fixed enum putUnknown for java etc.

**Comment by weidongxu-microsoft** [comment]
Another is possible wrong mockapi. I am still waiting this fix to be released to enable some tests https://github.com/Azure/cadl-ranch/pull/249

@haolingdong-msft But just start with things that works.

**Comment by haolingdong-msft** [comment]
> Another is some wrong mockapi. e.g. I am still waiting this fix to be released to enable some tests [Azure/cadl-ranch#249](https://github.com/Azure/cadl-ranch/pull/249)
> 
> @haolingdong-msft But just start with things that works.

of course, I will start on the workable ones.

**Comment by weidongxu-microsoft** [comment]
@haolingdong-msft 

Enums/Fixed/putUnknownValue

this is just compile error?

**Comment by haolingdong-msft** [comment]
> @haolingdong-msft
> 
> Enums/Fixed/putUnknownValue
> 
> this is just compile error?

Yes, in other word, we just don't allow passing unkown value to fixed enum: https://github.com/Azure/autorest.java/blob/main/typespec-tests/src/test/java/com/enums/fixed/FixedClientTest.java#L26

**Comment by haolingdong-msft** [comment]
Hi @XiaofeiCao, could you please try to add more cadl-ranch cases, https://azure.github.io/cadl-ranch/ this report shows the tests we don't supoprt yet.

**Comment by weidongxu-microsoft** [comment]
@haolingdong-msft please be more specific, as you are familiar with this. Please just list the row that you want Xiaofei to help.

**Comment by haolingdong-msft** [comment]
> @haolingdong-msft please be more specific, as you are familiar with this. Please just list the row that you want Xiaofei to help (and exclude things that known not work).
> 
> I know Dictionary -> NullableFloatValue does not work.

Update on  my comment above.

**Comment by weidongxu-microsoft** [comment]
I think I said Dictionary -> NullableFloatValue not work?

**Comment by weidongxu-microsoft** [comment]
Does Resiliency work?

**Comment by haolingdong-msft** [comment]
@weidongxu-microsoft  These are in `Not workable ones` list.

**Comment by weidongxu-microsoft** [comment]
Ok, so we had more not workable than TODO...

For Internal, anyone does this https://github.com/Azure/autorest.java/issues/2067 will add it.

**Comment by XiaofeiCao** [comment]
For Models/Property/Nullable, I'll wait for their release. Last release was 10 days ago.

I'll take [Internal](https://github.com/Azure/autorest.java/issues/2067) first.

**Comment by weidongxu-microsoft** [comment]
This one actually not added test https://github.com/Azure/cadl-ranch/tree/main/packages/cadl-ranch-specs/http/server/path

please wait for PR https://github.com/Azure/autorest.java/pull/2074, as cadl-ranch 0.11 changed some of it
